getting-started/consume-services.hbs.md

```diff
@@ -43,68 +32,115 @@ Before following this walkthrough, as app developer you must:
+    ```
+
+The `PARAMETERS` section is of particular interest as it lists the range of configuration options available to you when it comes to creating a claim for the given class.
+
+## <a id="stk-create-claim"></a> Create a claim for a service instance
+
+This section covers:
+
+* Using `tanzu service class-claim create` to create a claim for an instance of a class.
+* Using `tanzu service class-claim get` to get detailed information on the status of the claim.
+
+You can create a claim for an instance of a class using the `tanzu service class-claim create` command. Let's create a claim for the `rabbitmq-unmanaged` class. We'll pass a parameter to the command to set the storage capacity of the resulting instance to 3 Gigabytes, rather than using the default 1 Gigabyte.
 
     ```console
-    tanzu services class-claims list
+    tanzu service class-claim create rabbitmq-1 --class rabbitmq-unmanaged --parameter storageGB=3
     ```
 
     Expected output:
 
     ```console
-      NAME      CLASS     READY  REASON
-      rmq-1     rabbitmq  True   Ready
+      Creating claim 'rabbitmq-1' in namespace 'default'.
     ```
 
-1. Retrieve detailed information about the claim by running:
+You can get detailed information about the claim using the `tanzu service class-claim get` command.
 
     ```console
-    tanzu services class-claims get rmq-1
+    tanzu service class-claim get rabbitmq-1
     ```
 
     Expected output:
 
     ```console
-    Name: rmq-1
-    Namespace: default
-    Claim Reference: services.apps.tanzu.vmware.com/v1alpha1:ClassClaim:rmq-1
-    Class Reference:
-      Name: rabbitmq
-    Status:
-      Ready: True
-      Claimed Resource:
-        Name: rmq-1
-        Namespace: service-instances
-        Group: rabbitmq.com
-        Version: v1beta1
-        Kind: RabbitmqCluster
+      Name: rabbitmq-1
+      Namespace: default
+      Claim Reference: services.apps.tanzu.vmware.com/v1alpha1:ClassClaim:rabbitmq-1
+      Class Reference:
+        Name: rabbitmq-unmanaged
+      Parameters:
+        storageGB: 3
+      Status:
+        Ready: True
+        Claimed Resource:
+          Name: b5982046-a1e9-40cf-8282-00fe67a2f868
+          Namespace: default
+          Group:
+          Version: v1
+          Kind: Secret
     ```
 
-1. Record the value of `Claim Reference` from the previous command.
-This is the value to pass to `--service-ref` to create the application workload.
+It may take a moment or two for the claim to report `Ready: True`. Behind the scenes, the creation of the claim has triggered the on-demand creation of a helm release of the Bitnami RabbitMQ helm chart. Credentials and connectivity information required to connect to the RabbitMQ cluster have been formatted according to the [Service Binding Specification for Kubernetes](https://github.com/servicebinding/spec) and stored in a `Secret` in your namespace.
```

Comment:
```suggestion
It may take a moment or two for the claim to report `Ready: True`. Behind the scenes, the creation of the claim has triggered the on-demand creation of a Helm release of the Bitnami RabbitMQ Helm chart. Credentials and connectivity information required to connect to the RabbitMQ cluster have been formatted according to the [Service Binding Specification for Kubernetes](https://github.com/servicebinding/spec) and stored in a `Secret` in your namespace.
```